Workbooks/Azure Web Application Firewall/WAF monitoring/WAF Monitoring.workbook

```diff
@@ -0,0 +1,831 @@
+            "crossComponentResources": [
+              "{Workspaces}"
+            ],
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ]
+            },
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces",
+            "label": "WAF Type"
+          },
+          {
+            "id": "d54c1639-d46c-4655-9d76-d5416926a453",
+            "version": "KqlParameterItem/1.0",
+            "name": "WAF",
+            "label": "WAF Items",
+            "type": 2,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "let FakeData = (datatable (Message:string, ruleName_s:string, clientIp_s:string, clientIP_s:string, action_s:string, transactionId_s:string,trackingReference_s:string) [ \"\", \"\", \"\", \"\", \"\", \"\", \"\" ]);\r\nFakeData | union AzureDiagnostics\r\n| where (ResourceType == \"APPLICATIONGATEWAYS\" or ResourceType == \"FRONTDOORS\" or ResourceType == \"PROFILES\" or ResourceType == \"CDNWEBAPPLICATIONFIREWALLPOLICIES\") and (\"{WAFType:label}\" == \"All\" or (ResourceType == \"APPLICATIONGATEWAYS\" and \"{WAFType:label}\" contains \"application gateway\") or (ResourceType == \"FRONTDOORS\" and  \"{WAFType:label}\" contains \"azure front door\") or  (ResourceType == \"PROFILES\" and \"{WAFType:label}\" contains \"azure front door premium\") or (ResourceType==\"CDNWEBAPPLICATIONFIREWALLPOLICIES\" and \"{WAFType:label}\" contains \"cdn\"))\r\n| summarize Count=count() by Resource\r\n| order by Count desc, Resource asc\r\n| project Value = Resource, Label = strcat(Resource, \" - \", Count)",
```

Comment:
a bunch of these queries are unioning with table with a fake row, which has different columns (including requestUri_s which isn't in my table? but is used in the filtering? is this all intentional?), and that blank row is *always* filtered out because of the way the query is written?

```
a 
| union b 
| where c
| summarize d
```

is *NOT* the same as
```
a 
| union (b 
| where c) // parentheses are important here!
|  summarize by d
```

if i add parens to some of these queries i get a single row of 0 count for a blank row.

so either these queries need parens to keep the union empty row intact, OR they don't need the union with fakedata at all